src/customer.js

```diff
@@ -28,16 +28,16 @@ let roomData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/rooms/r
 
 Promise.all([bookingData, roomData]).then((requiredData) => {
   userID = parseInt(localStorage.getItem('userID'))
-  customer.rooms = requiredData[1];
   customer.booked = requiredData[0];
-  customer.myBookings(userID).forEach(room => {$('.bookings').append(`<p class="rooms">Room: ${room.roomNumber} ${room.date}</p>`)})
-  $('.spending').html(customer.totalSpend(customer.myBookings(userID)));
+  customer.rooms = requiredData[1];
+  customer.getMyBookings(userID).forEach(room => {$('.bookings').append(`<p class="rooms">Room: ${room.roomNumber} ${room.date}</p>`)})
+  $('.spending').html(customer.getTotalSpend(customer.getMyBookings(userID)));
 }).catch(data => console.log('Fetch error', data))
```

Comment:
`i tired refactoring to add the data when a new class is made but then it god wierd because the class wasnt global anymore and kinda got stuck`

I think this is talking about my comments for customer.js lines 31 and 21.
What if you declare your `let customer` on line 14 but don’t assign it yet.
Then you could assign it a value by instantiating it on line 31 and pass through (requiredData[0], requiredData[1] as arguments.
I think this would keep the class instantiation global and let you assign those properties from within the class itself.
Same login should apply for lines 32-34 in your manager.js file.